geode-assembly/build.gradle

```diff
@@ -320,8 +323,6 @@ distributions {
         exclude '**/gradlew.bat'
         exclude '**/gradle/wrapper/gradle-wrapper.jar'
         exclude '**/.gradle'
-        exclude '**/build/**'
```

Comment:
Because 'should be' isn't the same as 'is', and it matches on regular files named 'build' and 'out' also. `out` is a required file name of a concourse-resource, and it was being excluded by this line.

Reply:
Well that’s just downright a big in Gradle then. Any open tickets on the issue?

Reply:
Eh, I would rather find a way to keep the build file simple.  Instead of writing gradle code to work around this, how `bout changing the file names...?

Reply:
While I am not a fan of complicated builds I would rather keep the hacks for bugs closer to the source. In this case the bug is in Gradle so let's keep the fix in Gradle. Changing the Docker build to rename files to match the Concourse API would be making another build processes, removed from Gradle, responsible for Gradle's bug.